tests/dtm/test_rgw_s3_process_restart.py

```diff
@@ -680,6 +679,7 @@ def test_write_read_delete_during_rgw_restart(self):
                       "and parallel deletes during rgw restart:")
         write_proc = []
         w_args = {'bucket_prefix': self.bucket_name, 'object_prefix': self.object_name,
+                  'obj_size': self.test_cfg['size'],
```

Comment:
Should we add no. of loops also in params, so that read and delete processes would run for longer time.

Reply:
This will be done as part of different PR. As deletes are also to be removed from test.